x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts

```diff
@@ -8,61 +8,112 @@
 import { EntityDefinition, entityDefinitionSchema } from '@kbn/entities-schema';
 import { BUILT_IN_ID_PREFIX } from './constants';
 
-export const builtInServicesEntityDefinition: EntityDefinition = entityDefinitionSchema.parse({
-  id: `${BUILT_IN_ID_PREFIX}services`,
-  name: 'Services from logs',
-  type: 'service',
-  managed: true,
-  indexPatterns: ['logs-*', 'filebeat*'],
-  history: {
-    timestampField: '@timestamp',
-    interval: '1m',
-  },
-  latest: {
-    lookback: '5m',
-  },
-  identityFields: ['service.name', { field: 'service.environment', optional: true }],
-  displayNameTemplate: '{{service.name}}{{#service.environment}}:{{.}}{{/service.environment}}',
-  metadata: [
-    'data_stream.type',
-    'service.instance.id',
-    'service.namespace',
-    'service.version',
-    'service.runtime.name',
-    'service.runtime.version',
-    'service.node.name',
-    'service.language.name',
-    'agent.name',
-    'cloud.provider',
-    'cloud.instance.id',
-    'cloud.availability_zone',
-    'cloud.instance.name',
-    'cloud.machine.type',
-    'host.name',
-    'container.id',
-  ],
-  metrics: [
-    {
-      name: 'logRate',
-      equation: 'A / 5',
-      metrics: [
-        {
-          name: 'A',
-          aggregation: 'doc_count',
-          filter: 'log.level: *',
-        },
-      ],
+export const builtInServicesFromLogsEntityDefinition: EntityDefinition =
+  entityDefinitionSchema.parse({
+    id: `${BUILT_IN_ID_PREFIX}services_from_ecs_data`,
+    name: 'Services from ECS data',
+    description:
+      'This definition extracts service entities from common data streams by looking for the ECS field service.name',
+    type: 'service',
+    managed: true,
+    filter: '@timestamp >= now-10m',
+    indexPatterns: ['logs-*', 'filebeat*', 'apm*', 'metrics-*'],
```

Comment:
`apm-*` is a legacy index since 7.14 and probably shouldn't be queried. I expect most users we want to target have upgraded to 8.x and will therefore have metrics which are pre-aggregated and much cheaper to query, 

```suggestion
    indexPatterns: ['logs-*', 'filebeat*', 'metrics-*'],
```